tests/integration_tests/test_litellm.py

```diff
@@ -108,7 +108,7 @@ def test_litellm_openai_async():
     from litellm import litellm
 
     # from litellm import acompletion
-    guard = gd.Guard()
+    guard = gd.AsyncGuard()
```

Comment:
🤔 not sure this def doesnt work w the new async strats

Reply:
What doesn't work?

Reply:
asyncio.run( the guard( is throwing a error. does that need to be patched? i thought the 2 paths between async and not should be separate and not interoperable

Reply:
Right, I think we could afford some checks and error throwing, but we do not want to support passing an async llm callable to a sync guard